web/src/styles/index.scss

```diff
@@ -436,3 +436,10 @@ and (max-width: 600px) {
 .testimonials--offseason {
 	padding: 0 1.5rem 1.5rem;
 }
+
+.bmviii-app-font {
+    font-family: Cabin, serif;
```

Comment:
This font still isn't being loaded in for some reason

Reply:
Looks like this is just a CORS issue on non-chrome browsers, probably because google whitelists all its domains in its own browser? I'll try to find the right way to fix this...

Reply:
if non-chrome browsers are defaulting to serif, I can double check with design to see what a better default font should be

Reply:
See suggestion above for how to fix this